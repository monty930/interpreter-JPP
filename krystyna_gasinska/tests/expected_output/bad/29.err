interpreter: line 5, column 9 Return statement outside procedure