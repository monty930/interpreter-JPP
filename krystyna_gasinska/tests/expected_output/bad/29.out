in main g3: 1
in main g3: 2